src/main/cpp/subsystems/Elevator.cpp

```diff
@@ -6,81 +6,158 @@
 
 #include "subsystems/Elevator.h"
 #include "Neo.h"
+#include "Talon.h"
 
 namespace t34
 {
 
   Elevator::Elevator()
   : m_level(0)
-  , m_wrist_motor(998, SparkLowLevel::MotorType::kBrushless)
-  , m_vertical_motor_left(999)
-  , m_vertical_motor_right(1000)
-  , m_vertical_motors_pid(0.5, 0.0, 0.0)
-  , m_wrist_motor_pid(0.5, 0.0, 0.0)
+  , m_at_rest(true)
+  , m_algae_wrist_motor(997, SparkLowLevel::MotorType::kBrushless)
+  , m_coral_wrist_motor(998, SparkLowLevel::MotorType::kBrushless)
+  , m_left_motor(999)
+  , m_right_motor(1000)
+  , m_elevator_motors_pid(0.5, 0.0, 0.0)
+  , m_wrist_motors_pid(0.5, 0.0, 0.0)
+  , m_init_height(18_in)
+  , m_init_algae_angle(70_deg) //90 degrees is straight up, 0 degrees is horizontal. This wrist CAN NOT move greater than 70 degrees.
+  , m_init_coral_angle(0_deg) //0 degrees is straight down. Rotating to the left is positive change.
   {
-    m_vertical_motors_pid.SetTolerance(Neo::LengthToNEOUnit(0.5_in));
-    m_vertical_motor_right.Follow(m_vertical_motor_left);
+    m_elevator_motors_pid.SetTolerance(Neo::LengthToNEOUnit(0.5_in));
+    m_right_motor.Follow(m_left_motor);
   }
 
-  frc2::CommandPtr Elevator::MoveWristToCommand(units::degree_t angle)
+  frc2::CommandPtr Elevator::MoveWristToCommand(WristType wrist, units::degree_t angle)
   {
+    SparkMax* wrist_motor;
```

Comment:
Recommended practice: Always initialize your variables, especially pointers:

```suggestion
    SparkMax* wrist_motor{ nullptr };
```